src/compile-middleware.ts

```diff
@@ -66,6 +68,11 @@ function isSuccessful(response: Response) {
   return (statusCode >= 200 && statusCode < 300);
 }
 
+export const babelCompileCache = LRU<string>(<LRU.Options<string>>{
+  max: (10 * Math.pow(2, 20)),  // 10MB Cache Size
```

Comment:
A cli option would be fairly easy to add.  I'll accept the size in bytes so I don't have to translate it.